### ISSUE TITLE
BPTTUpdater AttributeError: can't set attribute

### ISSUE THREAD

**Issue by skelder** [Fhrozen/motion_dance#3]
**Describe the bug**

I am currently trying to run the `run.sh` on ubuntu 16.04 system.
I installed the repository using the `pip install .` command 
I also installed the marsyas following the their official guide
Until stage 2 everything is fine.
But once the code is running `stage 2: Training Network` I get `AttributeError: can't set attribute`

**To Reproduce**

from project main folder execute `./local/run.sh` without any arguments

**Screenshots**

![image](https://user-images.githubusercontent.com/10125862/85678506-860ceb80-b703-11ea-8473-d62e00b371bb.png)

**Desktop**
 - OS: Ubuntu 16.04

**Comment by Fhrozen** [comment]
Yes, thank you for the report. The problem is that the Updater on chainer 7.0 does not support device setup. 
I will fix this later.
Meanwhile, you can downgrade chainer to 6.4 or just comment on this line.
Thanks for the info
